src/Stratis.Features.SQLiteWalletRepository/Tables/DBTable.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Globalization;
+using NBitcoin;
+
+namespace Stratis.Features.SQLiteWalletRepository.Tables
+{
+    internal class DBTable
```

Comment:
Is it desirable to require table classes to inherit from this, or could it just be a static class? To me it doesn't make much sense to add inheritance just to access a static method.

Reply:
Done.